customer_outstanding_statement/views/statement.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="customer_outstanding_statement.statement_document">
+        <t t-call="web.external_layout">
+            <div class="page">
+                <div class="row">
+                    <div class="col-xs-5 col-xs-offset-7">
+                        <span t-field="o.name"/><br/>
+                        <span t-raw="o.contact_address.replace('\n\n', '\n').replace('\n', '&lt;br&gt;')"/>
+                        <span t-field="o.vat"/>
+                    </div>
+                    <h4 style="padding-left:20em">
+                        Outstanding Statement
+                    </h4>
+                    <p>
+                        Date: <span t-esc="Date[o.id]" /><br/>
+                        <t t-if="o.ref">Partner ref: <span t-field="o.ref"/></t>
+                    </p>
+
+                    <t t-if="Lines[o.id]">
+                        <br/>
+                        <t t-foreach="Lines[o.id]" t-as="currency">
+                            <br t-if="not currency_first" />
+                            <p>
+                                Outstanding Statement in <span t-esc="Currencies[o.id][currency].name"/>:
+                            </p>
+                            <table class="table table-condensed" style="border: 1px solid black; border-collapse: collapse;">
+                                <thead>
+                                    <tr>
+                                        <th style="border-right: 1px solid black;">Reference number</th>
+                                        <th class="text-center" style="border-right: 1px solid black;">Date</th>
+                                        <th class="text-center" style="border-right: 1px solid black;">Due Date</th>
+                                        <th style="border-right: 1px solid black;">Description</th>
+                                        <th class="text-right" style="border-right: 1px solid black;">Original Amount</th>
+                                        <th class="text-right" style="border-right: 1px solid black;">Open Amount</th>
+                                        <th class="text-right" style="border-right: 1px solid black;">Balance</th>
+                                    </tr>
+                                </thead>
+                                <tr t-foreach="Lines[o.id][currency]" t-as="line">
```

Comment:
@hveficent and here to as well a text as Aging Due report as of today.

Reply:
Ok